src/components/Modal/Modal.vue

```diff
@@ -75,6 +82,9 @@ export default {
   methods: {
     closeModal () {
       this.$emit('close');
+    },
+    getHeaderContent () {
+      return this.$slots.header()[0].children || 'modal dialog';
```

Comment:
What if the `header` slot is passed a set of elements, not text?  Maybe `title` should just be another `required` prop?

Reply:
I tried to avoid the breaking change as I thought it unlikely (in our current uses at least), but since we are adding 1 breaking change already, we may as well

Reply:
should it be required too?

Reply:
I couldn't find anything on it but I assumed Mercedes or Juan put it there when they wrote this
may I remove it?